fix three finger, one lifted bug

KryptKode opened the pull request:
**Notes**
-  the bug happened when three fingers are used on an image and one is lifted causing the  image to rotate and downscaled
- with this fix, the view only applies transformations on the view when more than 2 fingers are used. When one finger is lifted, no transformations are applied

tibbi commented on library/src/main/kotlin/com/davemorrissey/labs/subscaleview/SubsamplingScaleImageView.kt:
```diff
@@ -576,6 +578,10 @@ open class SubsamplingScaleImageView @JvmOverloads constructor(context: Context,
                 didRotateInGesture = false
                 preventRotatingInGesture = false
 
+                if(touchCount > 2){
```
add some spaces please, format the code

KryptKode replied:
Please check @tibbi

KryptKode committed:
fix three finger bug
- the bug happened when three fingers are used on an image and one is lifted causing the  image to rotate and downscaled

tibbi commented:
should be good, thanks
